src/open_inwoner/accounts/forms.py

```diff
@@ -43,11 +44,22 @@ class Meta:
             "email",
             "first_name",
             "last_name",
+            "phonenumber",
             "password1",
             "password2",
             "invite",
         )
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        # make phonenumber required when 2fa-sms login is enabled
+        config = SiteConfiguration.get_solo()
+        if config.login_2fa_sms is False:
```

Comment:
Let's keep it pythonic and not compare to booleans:

`if not config.login_2fa_sms:`